boards/arm/ip_k66f/pinmux.c

```diff
@@ -32,6 +37,15 @@ static int ip_k66f_pinmux_init(struct device *dev)
 	pinmux_pin_set(porta, 25, PORT_PCR_MUX(kPORT_MuxAsGpio));/* !ETH_PME */
 	pinmux_pin_set(porta, 26, PORT_PCR_MUX(kPORT_MuxAsGpio));/* !ETH_INT */
 #endif
+
+#if DT_HAS_NODE_STATUS_OKAY(DT_NODELABEL(spi1))
+	/* SPI1 CS0, SCK, SOUT, SIN - Control of KSZ8794 */
+	pinmux_pin_set(portb, 10, PORT_PCR_MUX(kPORT_MuxAsGpio));
```

Comment:
You could mux it to PCS, probably it is kPORT_MuxAlt2 (SPI1_PCS0), see also https://github.com/zephyrproject-rtos/zephyr/blob/master/boards/arm/frdm_k64f/pinmux.c#L106